ci/docker/codecov.Dockerfile

```diff
@@ -30,7 +30,7 @@ RUN spack repo rm --scope site dlaf && \
     spack -e ci concretize -f && \
     mkdir ${BUILD} && \
     ln -s ${BUILD} `spack -e ci location -b dla-future` && \
-    spack -e ci install --jobs ${NUM_PROCS} --keep-stage --verbose
+    spack -e ci --config "config:flags:keep_werror:all" install --jobs ${NUM_PROCS} --keep-stage --verbose
```

Comment:
It might be an overshoot, but I think it might be worth highlight it anyway.

We could have a folder with a common set of configurations, that we can include in all spack environments.

See https://spack.readthedocs.io/en/latest/environments.html#included-configurations

Reply:
Hmm, interesting. I'll see how easy it would be to update that in this PR still.

Reply:
Just saying, in case you go for it, we might also move other things that are already duplicated.

https://github.com/eth-cscs/DLA-Future/blob/bbc403cfe95872fc699e593ae3149b67dad0ed35/ci/docker/release-cpu.yaml#L19-L21

https://github.com/eth-cscs/DLA-Future/blob/bbc403cfe95872fc699e593ae3149b67dad0ed35/ci/docker/debug-cpu.yaml#L32-L44

Reply:
See https://github.com/eth-cscs/DLA-Future/pull/845 for the separate config file.

I'm marking this resolved based on the assumption that it's not worth creating a config file only for the installation of DLA-Future which needs `keep_werror:all`. Please unresolve if you disagree.